sr/comp/scorer/converter.py

```diff
@@ -0,0 +1,114 @@
+import copy
+import imp
+import sys
+from pathlib import Path
+from typing import cast, Dict, Type, Union
+
+from sr.comp.match_period import Match
+
+
+class Converter:
```

Comment:
Is the intention this serves as a base class, or a functional reference implementation? If the former i'd be interested in seeing it converted to `abc.ABC` with `abstractmethod`s

Reply:
Both. For trivial game designs it's reasonable that this might even be all that's needed.